src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_TupleLiteralExpression.vb

```diff
@@ -78,6 +81,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                     hasErrors:=True)
                     End If
 
+                    Binder.CheckRequiredMembersInObjectInitializer(tuple8Ctor, tuple8Ctor.ContainingType, initializers:=ImmutableArray(Of BoundExpression).Empty, syntax, _diagnostics)
```

Comment:
>Binder.CheckRequiredMembersInObjectInitializer(tuple8Ctor, tuple8Ctor.ContainingType, initializers:=ImmutableArray(Of BoundExpression).Empty, syntax, _diagnostics) [](http://example.com/codeflow?start=20&length=163)

Do we have a test where this line of code makes an observable difference?

Reply:
Yes, `TupleWithRequiredFields` has a tuple8Ctor with a required member.